armi/materials/air.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2022 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from armi.utils import units
+
+"""Simple air material"""
+
+from armi.materials import material
+from armi.utils.units import getTk
+
+
+class Air(material.Fluid):
+    """
+    Dry, Near Sea Level
+
+    Correlations based off of values in Incropera, Frank P., et al.
+    Fundamentals of heat and mass transfer. Vol. 5. New York: Wiley, 2002.
+
+    Elemental composition from PNNL-15870 Rev. 1
+            https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-15870Rev1.pdf
+    """
+
+    name = "Air"
+
+    """
+    temperature ranges based on where values are more than 1% off of reference
+    """
+    propertyValidTemperature = {
+        "density": ((100, 2400), "K"),
+        "heat capacity": ((100, 1300), "K"),
+        "thermal conductivity": ((200, 850), "K"),
+    }
+
+    def setDefaultMassFracs(self):
+        """
+        Set mass fractions
+
+        Notes
+        -----
+        Mass fraction reference McConn, Ronald J., et al. Compendium of
+        material composition data for radiation transport modeling. No.
+        PNNL-15870 Rev. 1. Pacific Northwest National Lab.(PNNL), Richland,
+        WA (United States), 2011.
+
+        https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-15870Rev1.pdf
+        """
+        self.setMassFrac("C", 0.000124)
+        self.setMassFrac("N", 0.755268)
+        self.setMassFrac("O", 0.231781)
+        self.setMassFrac("AR", 0.012827)
```

Comment:
The reason the tests are failing:

```
 The nuclides {'AR', 'N', 'O'} are present in material <Material: Air> by compositions, but are not specified in the `nuclide flags` section of the input file. 
```

I note we have `"O16"` in ARMI, so that's probably an easy direct-substitute.

But `"N"` amd `"AR"` are less obvious.

Reply:
I added a commit, to see if O16 nd N14 work. Not sure what to do about Argon. Yet.